packages/traceability-schemas/schemas/Observation.json

```diff
@@ -30,6 +30,12 @@
       "title": "Measured Value",
       "description": "The measurement of an Observation.",
       "$ref": "https://w3id.org/traceability/schemas/MeasuredValue.json"
+    },
+    "date": {
+      "$comment": "{\"term\": \"date\", \"@id\": \"https://schema.org/DateTime\"}",
```

Comment:
```
      "$comment": "{\"term\": \"date\", \"@id\": \"https://schema.org/DateTime\"}",
```
This is a *serious* misuse of `@id`, the value of which *should* be a unique-within-workspace identifier, *not* a datatype.  This PR may not be the best place to address this concern, but I thought it better to raise it now for redirection to the best place to address it, than to potentially forget about it.

Reply:
@TallTed these schemas are used to [generate the JSON-LD context](https://github.com/w3c-ccg/traceability-vocab/blob/gh-pages/contexts/traceability-v1.jsonld), in this case, I would expect to see a generated context that looks like this (after some corrections):

```
 "date": {
      "@id": "https://schema.org/observationDate",
      "@type": "http://www.w3.org/2001/XMLSchema#dateTime" // or...  https://schema.org/DateTime ... depending on your ISO8601 vs W3C XSD Date Time religious views...
    },
```

See a related example from the W3C VC Data Model:

https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v1#L100

Changing the value of `@id` would break signatures, we try and NOT define terms differently than schema.org, or some more reputable vocabulary than ourselves:

https://schema.org/Observation

Does my suggestion look better?

Reply:
Right.  My misread was because I forgot how `"$comment"` is being abused by holding `@context` definitions without `@context` being mentioned anywhere nearby... So I was reading this as data, not as `@context`.

Too many idiosyncrasies across all the docs I'm reading. :-/

Reply:
@OR13, I understand this now and certainly agree. 
But this goes beyond observationDate, doesn't it? The whole Observation schema should point at https://schema.org/Observation (not a re-invention of ours). Making this change also and we can discuss if needed.